OpenRA.Mods.Common/Traits/World/WeatherOverlay.cs

```diff
@@ -69,10 +69,19 @@ public class WeatherOverlayInfo : TraitInfo, ILobbyCustomRulesIgnore
 		[Desc("Works only with line enabled and can be used to fade out the tail of the line like a contrail.")]
 		public readonly byte LineTailAlphaValue = 200;
 
+		[Desc("How long the time the weather fades in when enabled.")]
```

Comment:
```suggestion
		[Desc("Time to fade in once the trait becomes enabled.")]
```

to be honest these names feel reverse as well

Reply:
fixed